Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Playwright/ActionHandlers/ActBrowserElementHandler.cs

```diff
@@ -0,0 +1,492 @@
+                if (content.StartsWith("<!DOCTYPE html>"))
+                {
+                    content = content.Substring("<!DOCTYPE html>".Length);
+                }
+                _act.AddOrUpdateReturnParamActual("PageSource", content);
+            }
+        }
+
+        private Task HandleCloseOperationAsync()
+        {
+            return _browser.CurrentWindow.CurrentTab.CloseAsync();
+        }
+
+        private async Task HandleCloseTabExceptOperationAsync()
+        {
+            if (string.IsNullOrEmpty(_act.ValueForDriver) && string.IsNullOrEmpty(_act.LocateValueCalculated))
+            {
+                throw new InvalidActionConfigurationException("Error: The window title to search for is missing.");
+            }
+
+            string excludedWindowTitle;
+            if (!string.IsNullOrEmpty(_act.LocateValueCalculated))
+            {
+                excludedWindowTitle = _act.LocateValueCalculated;
+            }
+            else
+            {
+                excludedWindowTitle = _act.ValueForDriver;
+            }
+
+            List<IBrowserTab> tabsToClose = [];
+            foreach (IBrowserWindow window in _browser.Windows)
+            {
+                foreach (IBrowserTab tab in window.Tabs)
+                {
+                    string tabTitle = await tab.GetTitleAsync();
+                    if (!string.IsNullOrEmpty(tabTitle) && tabTitle.Contains(excludedWindowTitle, StringComparison.OrdinalIgnoreCase))
+                    {
+                        continue;
+                    }
+                    tabsToClose.Add(tab);
+                }
+            }
+
+            Task[] tabCloseTasks = new Task[tabsToClose.Count];
+            for (int index = 0; index < tabsToClose.Count; index++)
+            {
+                tabCloseTasks[index] = tabsToClose[index].CloseAsync();
+            }
+            await Task.WhenAll(tabCloseTasks);
+        }
+
+        private Task HandleCloseAllOperationAsync()
+        {
+            List<IBrowserWindow> windows = new(_browser.Windows);
+            Task[] closeWindowTasks = new Task[windows.Count];
+            for (int index = 0; index < windows.Count; index++)
+            {
+                IBrowserWindow window = windows[index];
+                closeWindowTasks[index] = window.CloseAsync();
+            }
+            return Task.WhenAll(closeWindowTasks);
+        }
+
+        private Task HandleCheckPageLoadedOperationAsync()
+        {
+            return _browser.CurrentWindow.CurrentTab.WaitTillLoadedAsync();
+        }
+
+        private async Task HandleGetConsoleLogOperationAsync()
+        {
+            string logs = await _browser.CurrentWindow.CurrentTab.GetConsoleLogsAsync();
+            _act.AddOrUpdateReturnParamActual("Console logs", logs);
+        }
+
+        private async Task HandleGetBrowserLogOperationAsync()
+        {
+            string logs = await _browser.CurrentWindow.CurrentTab.GetBrowserLogsAsync();
```

Comment:
Add try catch to function to avoid crash, make sure to add it to all async function where logic is copmplex like this

Reply:
try/catch is present in HandleAsync method which is the parent of this and all the other async methods in this and ActUIElementHandler class.